llvmlite/ir/values.py

```diff
@@ -312,6 +311,64 @@ def __ne__(self, other):
     def __hash__(self):
         return hash(self.operands)
 
+class DIToken:
+    def __init__(self, value):
+        self.value = value
```

Comment:
The purpose of this class is to represent DWARF enumeration values that should appear bare in the debug information metadata.